src/Subsystems/Chassis.h

```diff
@@ -23,26 +12,21 @@ class Chassis: public Subsystem {
 private:
 	// It's desirable that everything possible is private except
 	// for methods that implement subsystem capabilities
-	// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
 	std::shared_ptr<DigitalInput> leftLightSensor;
 	std::shared_ptr<DigitalInput> rightLightSensor;
 	std::shared_ptr<CANTalon> leftFrontWheel;
 	std::shared_ptr<CANTalon> leftRearWheel;
 	std::shared_ptr<CANTalon> rightFrontWheel;
 	std::shared_ptr<CANTalon> rightRearWheel;
 
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
 
 	std::shared_ptr<RobotDrive> robotDrive;
 
```

Comment:
more spacing but from this point forward, I'm not going to address this individually.